src/Evolution/DiscontinuousGalerkin/Limiters/WenoOscillationIndicator.hpp

```diff
@@ -0,0 +1,33 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+/// \cond
+class DataVector;
+template <size_t>
+class Mesh;
+/// \endcond
+
+namespace Limiters {
+namespace Weno_detail {
+
+// Compute the WENO oscillation indicator (also called the smoothness indicator)
+//
+// The oscillation indicator measures the amount of variation in the input data,
+// with larger indicator values corresponding to a larger amount of variation
+// (either from large monotonic slopes or from oscillations).
+//
+// Implements an indicator similar to that of Eq. 23 of Dumbser2007, but with
```

Comment:
\cite Dumbser2008

Reply:
Does this make sense in non-Doxygen documentation?

Reply:
meh, okay